### PR TITLE
create internal/config/redis test

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:

I created the test for `internal/config/redis.go`.
grammar check has passed.

In `internal/config/redis.go`, I found `WithReadOnlyFlag()` is not used in `Opts()` method.
I'm not sure it is ok, please tell me how should I handle it ( @kpango or @rinx )

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/config/redis.go [hunk 1]
```diff
@@ -98,6 +99,8 @@ func (r *Redis) Bind() *Redis {
 	return r
 }
 
+// Opts creates the functional option list from the Redis.
+// If the error is ocurred, it will return no functional options and the error.
```
[the remark this answers is not in the document]
Comment (vankichi) [reply internal/config/redis.go:103]:
```suggestion
// If the error is occurred, it will return no functional options and the error.
```

**Review by hlts2 — APPROVED** [review]

**Review by kevindiu — DISMISSED** [review]
LGTM

**Comment by kevindiu** [comment]
@kpango @rinx can you please tell me should we handle the WithReadOnlyFlag() on opts? (see PR description)

**Comment by rinx** [comment]
I'm not sure but maybe at that point, passing the `readonly` flag to Redis sentinel client caused an error because it doesn't have that flag. So it was commented out temporarily.

**Comment by vankichi** [comment]
@kpango 
how about it?
Can we ignore it now?

**Review by hlts2 — DISMISSED** [review]

**Review by kevindiu — APPROVED** [review]
LGTM

**Comment by vankichi** [comment]
@kpango 
I added the comment about the reason why `WithReadOnlyFlg()` is disabled.
please check it, and if it is ok, plz merge it :pray:

**Review by kpango — DISMISSED** [review]

**Pull request merged**